subprojects/packagefiles/xtensor/meson.build

```diff
@@ -0,0 +1,42 @@
+project('xtensor', 'cpp',
+        version:'0.21.5',
+        license:'BSD-3-Clause',
+        default_options : ['warning_level=3', 'cpp_std=c++14'])
+
+xtl_dep = dependency('xtl',
+    version:['>=0.6.12', '<1.0.0'])
+
+json_dep = dependency('nlohmann_json',
+    version:'>=3.1.1')
+
+if get_option('use_xsimd')
+    xsimd_dep = dependency('xsimd',
+        version: ['>=7.4.6', '<8.0.0'])
+    xtensor_cpp_arg = ['-Duse_xsimd']
+else
+    xsimd_dep = dependency('', required: false) # aka "not found"
```

Comment:
We could really just use a feature option in meson_options.txt and then have dependency('xsimd') natively handle the =disabled case by skipping. Then key off of xsimd_dep.found()

Reply:
should be good

Reply:
That's not what I suggested at all.

Reply:
I don't really know what you were suggesting then. Seemed like what I changed it to is a simpler way of carrying out the same behavior as before

Reply:
> We could really just use a feature option in meson_options.txt

https://mesonbuild.com/Build-options.html#features

Reply:
Note that xsimd_dep would then always be added to the relevant targets, but if not found, or disabled, then it would be a no-op (adds no additional compile or link arguments).

Reply:
ohh..._now_ it should be good if I understand correctly

Reply:
Thank you! Yes, that does look much better now. Very beautiful. ❤️